internal/tui/model_list.go

```diff
@@ -36,16 +52,31 @@ type ListModel struct {
 	description string
 	suggestions []string
 	list        list.Model
+	input       textinput.Model
+	keys        keyMap
+	help        help.Model
 
 	confirmed bool
 	loading   bool
 
-	spinner  spinner.Model
-	selected string
+	spinner   spinner.Model
+	selected  item
+	usePrefix bool
 
 	err error
 }
 
+var listKeys = []key.Binding{
+	key.NewBinding(
+		key.WithKeys("t"),
+		key.WithHelp("t", "toggle prefix"),
+	),
+	key.NewBinding(
+		key.WithKeys("e"),
+		key.WithHelp("e", "edit suggestion"),
+	),
+}
```

Comment:
There is already a system for custom key bindings given by the `KeyMap()` function. I personally would prefer to add these key binds to that system.

Then you can use the keymap's `ShortHelp` function to provide help to the `AdditionalShortHelpKeys` function. And you can go back to using `kmap.Matches` for the update function.

DM me on slack for more info on how to filter specific keys out further, or if this is otherwise confusing.